Src/Plugins/Tests.Security/EndToEndTests.cs

```diff
@@ -104,8 +104,7 @@ protected override string ConstructTestOutputFromInputResource(string inputResou
                 {
                     TargetUri = new Uri(filePath, UriKind.Absolute),
                     FileContents = logContents,
-                    Logger = logger,
-                    MaxFileSizeInKilobytes = 1024,
```

Comment:
>MaxFileSizeInKilobytes = 1024 [](http://example.com/codeflow?start=20&length=29)

Since our default value changed to 10MB, we don't need to set this.